internal/workos/workos.go

```diff
@@ -2,5 +2,5 @@ package workos
 
 const (
 	// Version represents the SDK version number.
-	Version = "v4.9.0"
+	Version = "v4.10.0"
```

Comment:
are you an owner for the go sdk? i do it in 1 so that it doesn't require 2 owners checks (been blocked by this before)
but i'll grab whatever changes that were merged before this when i cut a release

Reply:
Yeah I'm a Gopher, so can approve the bump whenever.

Reply:
great, just undone